addons/im_livechat/static/src/core/web/transcript_sender_popover.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="im_livechat.TranscriptSenderPopover">
+        <ActionPanel title.translate="Send a copy of the conversation" icon="'fa fa-paper-plane'" resizable="false">
+            <div class="my-2 shadow-sm">
+                <TranscriptSender thread="props.thread" close="props.close"/>
```

Comment:
It doesn't realy makes sense to add a `close` props to the transcript sender itself (it's only an input and a button). I'd rather adding an optional callback prop `onSend` which will be used by the popover to close itself.